etc/river.toml

```diff
@@ -58,6 +58,8 @@ schema = "test"
 table = "t"
 index = "test"
 type = "t"
+# Default will use the mysql's primary key as es's id, if set id will use the id's column value as id
+id = ["id", "tags"]
```

Comment:
please use another rule section for this feature.
Btw, please add this to ReadMe too.